pkg/azurefileplugin/main.go

```diff
@@ -51,6 +51,7 @@ var (
 	mountPermissions                       = flag.Uint64("mount-permissions", 0777, "mounted folder permissions")
 	allowInlineVolumeKeyAccessWithIdentity = flag.Bool("allow-inline-volume-key-access-with-identity", false, "allow accessing storage account key using cluster identity for inline volume")
 	fsGroupChangePolicy                    = flag.String("fsgroup-change-policy", "", "indicates how the volume's ownership will be changed by the driver, OnRootMismatch is the default value")
+	experimentalEnableFsType               = flag.Bool("experimental-enable-fstype", false, "enable fstype storage class parameter usage (experimental)")
```

Comment:
we should set as `true` by default

Reply:
Done. I've also renamed the option so it better describes what it actually does.